wondrous-app/components/BreadCrumbs/Components/CollabSelector.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import { Separator } from 'components/Collaboration/SharedOrgHeader/styles';
+import { useMemo } from 'react';
+import palette from 'theme/palette';
+import typography from 'theme/typography';
+import { useGlobalContext } from 'utils/hooks';
+import { OrgComponent } from './OrgSelector';
+
+const CollabSelector = () => {
+  const { pageData } = useGlobalContext();
+  const activePodOrg = useMemo(() => pageData?.pod?.org, [pageData?.pod]);
+
+  const activeOrg = pageData?.orgData;
+
+  const collab = activeOrg || activePodOrg || {};
+
+  return (
+    <Grid display="flex" gap="14px" justifyContent="center" alignItems="center">
+      {collab?.parentOrgs?.map((org, idx) => (
```

Comment:
Nit: Technically don't need the ternary operator on collab

Reply:
true, fixed